sections/faqs/browser-support.js

```diff
@@ -0,0 +1,11 @@
+import md from "components/md"
+
+const BrowserSupport = () => md`
+  ## Which browsers does styled-components support?
+
+  Styled-components supports all the browsers that ReactDOM supports.
+  Please do read the [React docs section](https://reactjs.org/docs/react-dom.html#browser-support) to learn more.
```

Comment:
I think this sentence can be removed and instead we can add a link to "browsers that React supports" above.
It's perfectly clear that the link will contain a clarification 💯